L1Trigger/L1TMuonEndCap/python/simEmtfDigis_cfi.py

```diff
@@ -134,6 +134,7 @@
     CSCComparatorInput = cms.InputTag('muonCSCDigis','MuonCSCComparatorDigi'),
     RPCInput = cms.InputTag('muonRPCDigis'),
     CPPFInput = cms.InputTag('emtfStage2Digis'),
+    GEMInput = 'muonGEMPadDigiClustersReEmul',
```

Comment:
Shouldn't this also be `cms.InputTag('muonGEMPadDigiClustersReEmul')?

Reply:
Since `simEmtfDigisData` is a clone of an existing module which already defined those inputs, it's actually safer to drop the `cms.InputTag()` statements.

Reply:
ok I see. no problem then.

Reply:
And like I said, once the GEM clusters received by the EMTF at P5, we can consider switching to `emtfStage2Digis` from the EMTF unpacker.

Reply:
yes of course. I know this is temporary.